packages/website/src/features/Deploy/QueueFromGitOpsPage.tsx

```diff
@@ -194,15 +194,8 @@ export default function QueueFromGitOps() {
                 }
               />
             </div>
-            {/* Deployment alert status  */}
-            {selectedDeployType == 'git' && onChainPrevPkgQuery.isFetched && (
-              <PrevDeploymentStatus
-                prevDeployLocation={prevDeployLocation}
-                tomlRequiresPrevPackage={requiresPrevPackage}
-              />
-            )}
             {/* Hash: Prev deployment Input */}
-            {cannonDefInfo?.def && requiresPrevPackage && (
+            {cannonDefInfo?.def && !hasDeployers && (
```

Comment:
This is the proper fix for the bug. When we were not showing the upgrade-from input when needed.